MANIFEST

```diff
@@ -5104,6 +5104,7 @@ Makefile.SH			A script that generates Makefile
 malloc.c			A version of malloc you might not want
 malloc_ctl.h			A version of malloc you might not want
 MANIFEST			This list of files
+MANIFEST.SKIP			Patterns for files not included in this file
```

Comment:
was wondering if this should belong to the MANIFEST.SKIP :-)

Reply:
Not in this branch, perhaps in a latter once we have actually defined what we do and do not want in the MANIFEST.

Reply:
sounds fair